lib/functions/HttpUtils.ts

```diff
@@ -19,15 +19,15 @@ export async function getJson<T>(URL: string): Promise<T> {
   } else {
     if (response.status && response.status < 400) {
       const json = await response.json();
-      console.log(`Well-knonw response: ${JSON.stringify(json, null, 2)}`);
+      // console.log(`Well-knonw response: ${JSON.stringify(json, null, 2)}`);
```

Comment:
Remove commented out code

Reply:
No please don't.  We will do that at the end. We really need to be able to see responses and/or enable comments at this stage

Reply:
reverted that